core/shared/config/env/config.development.json

```diff
@@ -22,5 +25,23 @@
         "admin": {
             "maxAge": 0
         }
+    },
+    "url": "https://ghost.localhost/blog",
+    "admin":{
+        "url": "https://ghost-admin.localhost/blog"
+    },
+    "mail": {
+        "from": "support@example.com",
+        "transport": "SMTP",
+        "options": {
+            "service": "Mailgun",
+            "host": "localhost",
+            "port": 1025,
+            "secure": false,
+            "auth": {
+                "user": "username",
+                "pass": "password"
+            }
+        }
```

Comment:
@tpatel Guess this change was accidental and meant for `config.local.json` ?

Reply:
huh good catch 🙈

Reply:
@rishabhgrg It's fixed ✅